lib/trento_web/router.ex

```diff
@@ -75,6 +75,8 @@ defmodule TrentoWeb.Router do
       get "/sap_systems/health", HealthOverviewController, :overview
       get "/databases", SapSystemController, :list_databases
 
+      delete "/hosts/:id", HostController, :delete
```

Comment:
Can you move it under the /hosts/id/tags? so we can have visually grouped routes